src/style.css

```diff
@@ -1,3 +1,84 @@
 body {
+  --header-heigth: 60px;
+
+    margin: 0;
+    font-family: Roboto, sans-serif;
+    background-color: grey;
+    position: relative;
+  }
+
+.header {
+  position: relative;
+  background-color: #fff;
   margin: 0;
+  display: flex;
+  justify-content: space-between;
+  min-width: 1024px;
+  max-width: 1200px;
+  width: 100%;
+}
+
+.logo {
+  position: relative;
+  display: flex;
+  width: 40px;
+  height: 40px;
+  margin: 0 0 0 50px;
+  align-self: center;
+}
+
+
+.nav {
+  position: relative;
+  display: flex;
+  padding: 0;
+  margin: 0 50px 0 0;
+}
+
+.ul {
+  position: relative;
+  display: flex;
+  list-style: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  margin: 0;
+  padding: 0;
+}
+
+.li_items {
+  margin-right: 20px;
+  height: var(--header-heigth);
+}
+
+li:last-child {
+  margin: 0;
+}
+
+.a_items {
+  color: #000;
+  display: flex;
+  text-decoration: none;
+  position: relative;
+  align-items: center;
+  height: var(--header-heigth);
+}
+
+.item-1 {
+  color: #00ACDC;
+}
+
+.a_item4:hover {
+  position: relative;
+  color: #00ACDC;
+}
```

Comment:
- Every link should change the color on hover
- ![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/d42507fd-adf4-4d9b-98e3-d416117d952f)